### PR TITLE
Switch to Perlbrew to obtain multiple Perl versions

### PR DESCRIPTION
**Pull request by kjsanger**
Change some file names and labels to be consistent with other Perl CI

### CODE REVIEW

File: .github/workflows/run-tests.yml [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+          conda config --prepend pkgs_dirs ~/conda/pkgs
+          conda config --prepend envs_dirs ~/conda/envs
+
+          conda config --set auto_update_conda False
+          conda config --prepend channels "$WSI_CONDA_CHANNEL"
+          conda config --append channels conda-forge
+          conda info
+
+          conda create  -y -n "$CONDA_TEST_ENV"
+          conda install -y -n "$CONDA_TEST_ENV" biobambam2
```
Comment (mgcam) [inline .github/workflows/run-tests.yml:61]: Cannot see biobambam used in the package. Shall we try removing this?

File: .github/workflows/run-tests.yml [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+        run: |
+          cpanm --local-lib=${{ env.PERL_CACHE }} local::lib
+          eval $(perl -I ${{ env.PERL_CACHE }}/lib/perl5/ -Mlocal::lib="$NPG_LIB")
+          eval $(perl -I ${{ env.PERL_CACHE }}/lib/perl5/ -Mlocal::lib)
+          
+          cpanm --quiet --notest Module::Build
+          cpanm --quiet --notest Alien::Tidyp
+          cpanm --quiet --notest Net::SSLeay
+          cpanm --quiet --notest LWP::Protocol::https
+          cpanm --quiet --notest https://github.com/chapmanb/vcftools-cpan/archive/v0.953.tar.gz
```
Comment (mgcam) [inline .github/workflows/run-tests.yml:98]: This is unlikely to be needed, probably was left over from some much older version of the package.

File: .github/workflows/run-tests.yml [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+          
+          ./scripts/install_wsi_dependencies.sh "$NPG_LIB" \
+             perl-dnap-utilities \
+             ml_warehouse \
```
Comment (mgcam) [inline .github/workflows/run-tests.yml:102]: ml_warehouse dependency is likely to be redundant

**Review by mgcam — CHANGES_REQUESTED** [review]
Would it be possible to remove what I believe to be redundant dependencies or at least try a build without them?

File: .github/workflows/run-tests.yml (hunk 1 above)
Comment (kjsanger) [reply .github/workflows/run-tests.yml:61]: I tried removing it, but it's used by seqchksum_merge.pl

File: .github/workflows/run-tests.yml (hunk 3 above)
Comment (kjsanger) [reply .github/workflows/run-tests.yml:102]: Looking at `Build.PL`, it requires `npg_tracking` and that requires `ml_warehouse`.

Comment (mgcam) [reply .github/workflows/run-tests.yml:102]: Ah

File: .github/workflows/run-tests.yml (hunk 2 above)
Comment (kjsanger) [reply .github/workflows/run-tests.yml:98]: Sorry - that's just me being confused by GitHub's poor UI.

**Pull request merged**